backend/src/main/java/com/woowacourse/kkogkkog/member/application/MemberService.java

```diff
@@ -59,6 +59,15 @@ public Long save(SlackUserInfo userInfo, Workspace workspace, String nickname) {
         return newMember.getId();
     }
 
+    public Long save(GoogleUserInfo userInfo, String nickname) {
+        String email = userInfo.getEmail();
```

Comment:
슬랙 OAuth 때도 느낀 점이지만 특별한 이유가 없다면 클래스명이 굳이 ~Info, ~Data로 끝날 이유는 없다고 생각해요. 
데이터를 전달한다는 취지를 강조하고자 한다면 ~Dto라고 하는건 어떨까요?

```suggestion
    public Long save(GoogleUser userInfo, String nickname) {
        String email = userInfo.getEmail();
```

Reply:
저도 정의 의견에 어느정도 동의합니다. userInfo는 oauth 규약도 아니고 하는 역할도 dto와 비슷한 수준이라고 생각해서 `GoogleUserDto` 정도로 가는것도 일리있다고 생각합니다.

가능할지 모르겠지만, 만약 추상화를 진행하게 된다면 Dto라는 이름보다는 적절한 이름이 필요하지 않을까요?

Reply:
추상화가 적절할지는 코드를 봐야 알 것 같은데, 
함께 묶는다면 OAuthUser를 상위타입으로 하고, SlackUser, GoogleUser라는 구현체를 만드는건 어떨까요?

Reply:
저도 이름만 적절하게 통일된다면 어떤 명칭을 사용하든 문제는 없다고 생각해요!

Reply:
GoogleUser는 마치 저희 Member인것 처럼 보일 수 있어서 일단 GoogleUserDto로 변경하겠습니다